core/block.go

```diff
@@ -21,6 +21,11 @@ type Header struct {
 	Number uint64
 	// The state commitment after this block
 	GlobalStateRoot *felt.Felt
+
+	TransactionCommitment *felt.Felt
+
+	EventCommitment *felt.Felt
+
```

Comment:
we do have these in a separate DB bucket (`BlockCommitments`). Let's not add them here for now. It might make sense to fold them in to `core.Header` in a separate PR with a migration.